apps/api/src/app/inbox/e2e/get-preferences.e2e.ts

```diff
@@ -68,6 +69,14 @@ describe('Get all preferences - /inbox/preferences (GET)', function () {
       noFeedId: true,
       tags: [securityTag],
     });
+    await session.createTemplate({
+      noFeedId: true,
+      tags: [marketingTag],
+    });
+    await session.createTemplate({
+      noFeedId: true,
+      tags: [],
+    });
```

Comment:
2 more workflows are creating to ensure the assertion accounts for both:
* workflows with mismatched tags
* workflows without tags